swan-cern/values.yaml

```diff
@@ -268,29 +268,30 @@ fluentd:
           path: ca-bundle.crt
 
 gpu-operator:
-  enabled: true
+  enabled: false
   validator:
     repository: registry.cern.ch/kubernetes
-    version: v22.9.1
+    version: v24.6.2
   operator:
     repository: registry.cern.ch/kubernetes
-    version: v22.9.1
+    version: v24.6.2
     defaultRuntime: containerd
     initContainer:
       repository: registry.cern.ch/kubernetes
       version: 11.4.2-base-ubi8
   driver:
     repository: registry.cern.ch/kubernetes
     image: nvidia-gpu-driver
-    version: "v550.54.15-6.6.13-200.fc39.x86_64"
+    version: "v550.54.15-6.8.4-200.fc39.x86_64"
     imagePullPolicy: Always
     manager:
       repository: registry.cern.ch/kubernetes
       version: v0.6.0
     licensingConfig:
       configMapName: "nvidia-grid-license"
-    kernelModuleConfig:
-      name: kernel-module-params
+      nlsEnabled: false
+  nfd:
```

Comment:
These changes are also unrelated to the update to fix the security issue? And also others below.

Reply:
It is to keep our chart in sync with the cern-magnum one. Should they be removed?

Reply:
What we can do, not to complicate things too much, is to separate this in two commits: one that updates the chart up until before the security issue fix, and one that strictly has the changes to fix the security issue.

Reply:
Yeah that sounds good